specs/darwin/process_es_events.table

```diff
@@ -0,0 +1,32 @@
+table_name("process_es_events")
```

Comment:
We might want to use `es_process_events` since BPF set this "prefix" precedent already.

Reply:
This began life as `es_process_events` :)
But I gather from comments/office hours, that it would be nice to group `process_events` and `process_es_events` together (for when someone is browsing all available tables). 

I am not set any way, and if BPF has set a precedent, I am happy to change it, just let me know!
The only restriction is that the table name ends with `_events` since we have that hardcoded in our codebase (in `core` and `amalgamation` iirc)

Reply:
Yeap, I remember the previous preference and I think we should stick with the BPF precedent for now.

It's easy to rename and alias them all later on.

Reply:
Sounds good! Updated